promql/engine.go

```diff
@@ -2232,6 +2239,54 @@ func (ev *evaluator) VectorscalarBinop(op parser.ItemType, lhs Vector, rhs Scala
 	return enh.Out
 }
 
+// RangeVectorscalarbinop evaluates a binary operation between a RangeVector and a Scalar.
+func (ev *evaluator) RangeVectorscalarbinop(op parser.ItemType, lhs, rhs parser.Expr, swap bool) (Matrix, storage.Warnings) {
+	var warnings storage.Warnings
```

Comment:
Did we miss proper camel case in the function name:
RangeVectorscalarbinop --> RangeVectorScalarBinop

Reply:
Or even RangeVectorScalarBinOp